Peer.Connectors/AzdoPullRequestSource.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.Core.WebApi;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Peer.Domain;
+using Peer.Domain.Models;
+using Peer.User;
+
+namespace Peer.Connectors
+{
+    public class AzdoPullRequestSource : IPullRequestSource<AzdoPullRequestSource>
+    {
+        private readonly AppConfig _config;
+        private static IClient<UserAzdoClient> _userClient;
+
+        public AzdoPullRequestSource(AppConfig config, IClient<UserAzdoClient> userClient)
+        {
+            _config = config;
+            _userClient = userClient;
+        }
+
+        public async Task<IEnumerable<PeerPullRequest>> FetchPullRequests()
+        {
+
+            var userclient = _userClient.CreateClient();
+            var AzClient = userclient.AzClient;
+            var AzProjClient = userclient.AzProjClient;
+
+            var projects = await AzProjClient.GetProjects();
+
+            List<GitPullRequest> prs = new();
```

Comment:
much more common to see var prs = new List<GitPullRequest>() but dealer's choice here

Reply:
done